service/lib/ops/utils.js

```diff
@@ -8,6 +9,13 @@ function boomErrorWrapper (next) {
   }
 }
 
+function isDuplicationIdError (err, table) {
+  let errorMessage = `duplicate key value violates unique constraint "${table}_id_key"`
```

Comment:
relying on the actual error message seems fragile and prone to errors (different pg versions on different locales could lead to this being outdated)

I think it would be best to reply on the actual error object and error codes. See here for further details: https://www.postgresql.org/docs/9.3/static/errcodes-appendix.html

> For some types of errors, the server reports the name of a database object (a table, table column, data type, or constraint) associated with the error; for example, the name of the unique constraint that caused a unique_violation error. Such names are supplied in separate fields of the error report message so that applications need not try to extract them from the possibly-localized human-readable text of the message

Error code here should be `23505: unique_violation`